README.md

```diff
@@ -11,19 +11,36 @@ and attacher for Ceph RBD and CephFS.
 
 ## Overview
 
-Ceph CSI plugins implement an interface between CSI enabled Container
-Orchestrator (CO) and CEPH cluster.
-It allows dynamically provisioning CEPH volumes and attaching them to
-workloads.
-Current implementation of Ceph CSI plugins was tested in Kubernetes
-environment (requires Kubernetes 1.13+), but the code does not rely on
-any Kubernetes specific calls (WIP to make it k8s agnostic) and
-should be able to run with any CSI enabled CO.
+Ceph CSI plugins implement an interface between CSI enabled Container Orchestrator
+(CO) and CEPH cluster. It allows dynamically provisioning CEPH volumes and
+attaching them to workloads.
 
-For details about configuration and deployment of RBD and
-CephFS CSI plugins, see documentation in `docs/`.
+Independent CSI plugins are provided to support RBD and CephFS backed volumes,
 
-For example usage of RBD and CephFS CSI plugins, see examples in `examples/`.
+- For details about configuration and deployment of RBD and CephFS CSI plugins,
+  see documentation in `docs/`.
+- For example usage of RBD and CephFS CSI plugins, see examples in `examples/`.
+
+## Project status
+
+Status: **Alpha**
+
+The **alpha** status reflects possible non-backward compatible changes in the
+future, and is thus not recommended for production use. There is work in progress
+that would change on-disk metadata for certain operations, possibly breaking
+backward compatibility.
+
+## Supported CO platforms
+
+Ceph CSI drivers are currently developed and tested **exclusively** on Kubernetes
+environments. There is work in progress to make this CO independent and thus
+support other orchestration environments in the future.
+
+For Kubernetes versions 1.11 and 1.12, please use [0.3 images and
+deployments](https://github.com/ceph/ceph-csi/tree/csi-v0.3/deploy/).
```

Comment:
@ShyamsundarR I think we should not mentioned about `0.3` version any more, that version is deprecated in upstream.

Reply:
#315 still carries this version, I do not understand when this was deprecate?

Reply:
I think its fine to carry `0.3`, `0.2` is the deprecated one .